src/stlinit.c

```diff
@@ -260,6 +260,7 @@ stl_read(stl_file *stl, int first_facet, int first) {
   float *facet_floats[12];
   char facet_buffer[12 * sizeof(float)];
   uint32_t endianswap_buffer;  /* for byteswapping operations */
+  float read_count = 0;  /* for checking ASCII .stl read result */
```

Comment:
I am a bit confused. Why is this a float?

Reply:
That's a mistake, I'm so sorry.

Reply:
Do problem. I was just not sure.